App.js

```diff
@@ -40,8 +46,10 @@ class App extends Component {
       secretKey: md5(DeviceInfo.getUniqueID())
     });
 
+    store = configureStore();
```

Comment:
I'm curious, any specific reason to do this in here instead inside `persistStore(...)`?